saleor/core/context_processors.py

```diff
@@ -19,8 +19,10 @@ def default_currency(request):
 
 # request is a required parameter
 # pylint: disable=W0613
-def categories(request):
-    return {'categories': Category.tree.root_nodes()}
+def navigation(request):
+    menus = Menu.objects.prefetch_related(
+        'items', 'items__collection', 'items__category', 'items__page').all()
```

Comment:
if you prefetch `foo__bar` then `foo` will be automatically prefetched

Reply:
Right, I'll remove `items` from here.